qa/tasks/cephfs/test_sessionmap.py

```diff
@@ -215,3 +216,38 @@ def test_session_reject(self):
         with self.assert_cluster_log("client session with non-allowable root '/baz' denied"):
             with self.assertRaises(CommandFailedError):
                 self.mount_b.mount(mount_path="/foo/bar")
+
+    def test_session_evict_blacklisted(self):
+        """
+        Check that mds evicts blacklisted client
+        """
+        self.fs.set_max_mds(2)
```

Comment:
We to raise `SkipTest` if the client is the kernel:

/ceph/teuthology-archive/pdonnell-2019-06-04_03:04:48-kcephfs-wip-pdonnell-testing-20190603.231819-distro-basic-smithi/4003069/teuthology.log